### ISSUE TITLE
B905 should not be raised for itertools.repeat

### ISSUE THREAD

**Issue by nonamethanks** [PyCQA/flake8-bugbear#378]
```python
import itertools
list(zip([1, 2, 3], itertools.repeat("a")))
```

This should not raise a B905. `repeat` is by design infinite, that's the entire reason why it exists.

**Comment by NewGlad** [comment]
can be closed

**Comment by cooperlees** [comment]
Thanks!
